### DISCUSSION TITLE
Find the cursor position

### DISCUSSION THREAD

**Discussion by SoilihBenSoilih** [ocaml/ocaml-lsp#566] [Q&A]
Hello,

I'm using Lsp library. I want to find the cursor position in a file.
I have some informations like **uri**, **file content**. Is there a way to
do that using something available in the library ?

Thank you !

**Comment by ulugbekna** [comment]
I don’t think you can do that. The client needs to provide that info, but
you can’t ask the client to do so unless you can a custom request and
handle that both on the client and server

**Reply by SoilihBenSoilih** [reply]
Ooh ok, I see.

Thank you for the replay !
